app/assets/javascripts/components/star_charts_page.js

```diff
@@ -186,7 +210,7 @@ $(function() {
         dom.div({}, dateRangeText),
         dom.div({},
           dom.span({}, 'Students: ', props.students.length),
-          dom.span({ style: { paddingLeft: 10 }}, 'Data points: ', (props.assessments || []).length)));
+          dom.span({ style: { paddingLeft: 10 }}, '# Data points: ', (props.assessments || []).length)));
```

Comment:
Using the symbol might be cryptic, how about "Number of data points" if that fits?